### ISSUE TITLE
Installer doesn't work with screen readers

### ISSUE THREAD

**Issue by adhodgson1** [openaudible/openaudible#183]
It appears the installer doesn't provide feedback to screen readers, although the OpenAudible application works well.  This means that blind people using speech output technology such as Jaws for Windows, NVDA etc cannot install OpenAudible independantly or have to follow specific instructions.

Workarounds:
- Use Chocolatey to install the OpenAudible package.
- Use a guided instlalation method such as: https://lists.bcab.org.uk/pipermail/bcab/2019-August/095218.html

**Comment by openaudible** [comment]
I'm going to close this with the accepted answer to run the installer from the command line as:

.\OpenAudible_win_1.5.3.exe -q -overwrite
